docs/framework-introscope_agent.md

```diff
@@ -24,9 +24,8 @@ The credential payload of the service may contain the following entries:
 | Name | Description
 | ---- | -----------
 | `agent-name` | (Optional) The name that should be given to this instance of the Introscope agent
-| `host-name` | The host name of the Introscope Enterprise Manager server
-| `ssl` | (Optional) Whether or not to use an SSL connection to the Introscope Enterprise Manager server
-| `port` | (Optional) The port of the Introscope Enterprise Manager server
+| `url` | The url of the Introscope Enterprise Manager server
```

Comment:
Are there any existing Service Brokers that aren't going to be compatible with this?  I don't think that many customers were using the Introscope agent, but it's worth thinking about whether breaking this incompatibly is going to harm any current users.